micro-rdk-installer/src/main.rs

```diff
@@ -0,0 +1,96 @@
+use std::fs::{self, File};
+use std::io::{stdin, stdout, Write};
+
+use clap::{arg, command, Args, Parser, Subcommand};
+use micro_rdk_installer::nvs::data::{ViamFlashStorageData, WifiCredentials};
+use micro_rdk_installer::nvs::partition::{NVSPartition, NVSPartitionData};
+use micro_rdk_installer::nvs::request::populate_nvs_storage_from_app;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+struct AppCloudConfig {
+    r#id: String,
+    app_address: String,
+    secret: String,
```

Comment:
might consider using something like (secrecy)[https://crates.io/crates/secrecy] to keep debug and other outputs from printing out secrets accidentally.

Reply:
good catch, this is a very necessary change